app/models/game.rb

```diff
@@ -2,4 +2,9 @@ class Game < ActiveRecord::Base
   has_many :matches
   has_many :users, through: :matches
   has_many :chess_pieces
+
+
+  def piece_in_square?(x,y)
+    chess_piece.find_by position_x: x, position_y: y
```

Comment:
game `has_many` pieces therefore the method to get all of the pieces for the game is `chess_pieces`, vice versa a piece can only `belong_to` one game, therefore `game` is singular.